rpcd/etc/openstack_deploy/README.rst

```diff
@@ -0,0 +1,46 @@
+The user variables files for RPCO are broken out into the following:
+
+-  ``user_osa_variables_defaults.yml`` - This file contains any
+   variables specific to the OpenStack-Ansible project that are
+   overridden and are now the default values for RPCO deployments. The
+   values defined in here will be used for RPCO deployments by default.
```

Comment:
This sentence is basically saying the same thing as the previous sentence.  Total nit, but it could come out.